lib/acts_as_list/active_record/acts/position_column_method_definer.rb

```diff
@@ -49,6 +51,25 @@ def self.define_class_methods(caller_class, position_column, touch_on_update)
 
       private
 
+      # Update recordset to start at the given position
+      define_singleton_method :shuffle_to do |position|
+        max = unscoped.maximum(position_column)
```

Comment:
I don't like using `unscoped` without restrictions as it causes havoc for end-users. Just unstop what you need. Check out our other usages of `unscope`.

Reply:
I'll add support for rails <4 per the other cases. In this case, it's finding a 'safe' value for `position_column` that definitely can't conflict, as documented in the readme (line 108).